cvat-core/src/server-proxy.ts

```diff
@@ -1420,7 +1420,23 @@ async function getData(tid, jid, chunk) {
     return response;
 }
 
-async function getMeta(session, jid) {
+export interface FramesMetaData {
+    chunk_size: number;
+    deleted_frames: number[];
+    frame_filter: string;
+    frames: {
+        width: number;
+        height: number;
+        name: string;
+        related_files: number;
+    }[];
+    image_quality: number;
+    size: number;
+    start_frame: number;
+    stop_frame: number;
+}
+
```

Comment:
Circular dependency will appear in this case because frames.ts imports server-proxy.ts.
We need to define a separated files with types in the future when working on typification cvat-core.